pkg/kubelet/kubelet.go

```diff
@@ -1049,9 +1128,19 @@ func (kl *Kubelet) syncPod(pod *api.BoundPod, dockerContainers dockertools.Docke
 				return err
 			}
 		}
+		podStatus, err = kl.GetPodStatus(podFullName, uid)
```

Comment:
Yeah, the thing is I'll remove it from one branch in the other refactor and it'll be necessary. I wanted to move it to this PR to make the cut 'cleaner'.